corehq/apps/case_importer/views.py

```diff
@@ -32,6 +33,14 @@ def render_error(request, domain, message):
     return HttpResponseRedirect(base.ImportCases.get_url(domain=domain))
 
 
+def validate_column_names(column_names, invalid_column_names):
+    for column_name in column_names:
+        try:
+            lxml.etree.Element(column_name)
```

Comment:
FYI I found this function (I did look for it before but didn't find it then): https://github.com/dimagi/commcare-hq/blob/24c7cfc5fb5246dac52b94d318af870ee99a7667/corehq/apps/app_manager/models.py#L5420

It allows `/` so wouldn't work as is here but it wouldn't be hard to modify. It probably makes sense to use the same validation in both places.